stingray/tests/test_lightcurve.py

```diff
@@ -325,7 +324,8 @@ def test_add_with_different_err_dist(self):
                          err_dist="gauss")
         with warnings.catch_warnings(record=True) as w:
             lc = lc1 + lc2
-            assert "ightcurves have different statistics" in str(w[0].message)
+            assert np.any(["ightcurves have different statistics"
+                           in str(wi.message) for wi in w])
```

Comment:
I feel like there should probably be an "L" preceding "ightcurves"? :D

Reply:
I get stressed with capitalized letters, I try to avoid them :D